src/viur/core/modules/user.py

```diff
@@ -336,18 +336,9 @@ def login(self, *, name: str | None = None, password: str | None = None, **kwarg
         # next, check if the password hash matches
         is_okay &= secrets.compare_digest(password_data.get("pwhash", b"-invalid-"), password_hash)
 
-        # next, check if the user account is active
-        is_okay &= (user_skel["status"] or 0) >= Status.ACTIVE.value
```

Comment:
But where will the status check at login be in future? I only see the one at pwrecover and this one has been removed.
It should then either be moved to `next_or_finish` or remain here.

Reply:
Hi @sveneberth, regardless of the authentication method, the user's status is always checked in [`User.authenticateUser()`](https://github.com/viur-framework/viur-core/blob/develop/src/viur/core/modules/user.py#L1397).

This check here is redundant now, and a relict of the first ViUR version.

Reply:
Okay, I don't know why I overlooked `authenticateUser` earlier ...